fms/models/__init__.py

```diff
@@ -0,0 +1,50 @@
+from contextlib import nullcontext
+import os
+from pathlib import Path
+import re
+from typing import Callable, List
+import torch
+from torch import nn
+
+__models = {}
+
+
+def register_model(architecture, variant, factory):
```

Comment:
Can you include docstrings here as a user could be using this if they have their own model type they would like to register